spec/controllers/line_items_controller_spec.rb

```diff
@@ -32,9 +32,13 @@
       order = create(:completed_order_with_totals)
       item = create(:line_item, order: order)
       while !order.completed? do break unless order.next! end
+      order.reload
```

Comment:
We don't need to `reload` that is just to replicate the issue we are seeing on the upgrade branch